examples/step-10/step-10.cc

```diff
@@ -30,10 +30,14 @@
 #include <deal.II/grid/grid_out.h>
 #include <deal.II/dofs/dof_handler.h>
 #include <deal.II/dofs/dof_accessor.h>
-#include <deal.II/fe/fe_nothing.h>
 #include <deal.II/fe/fe_values.h>
 
-// This is the only new one: in it, we declare the MappingQ class
+// This one is new. In this tutorial, since we are not solving a PDE, we will
+// use include the below header file which provides the dummy class FE_Nothing
+// that has zero degrees of freedom (this will be explained later)
```

Comment:
This sounds rather convoluted. What about:
> // This include file is new. Even if we are not solving a PDE in this tutorial, we want to
// use a dummy finite element with zero degrees of freedoms provided by the FE_Nothing class.

Reply:
Perfect. I will update the PR by committing these changes.

Reply:
Done.